base/server/python/pki/server/cli/range.py

```diff
@@ -258,3 +259,165 @@ def execute(self, argv):
             sys.exit(1)
 
         subsystem.update_ranges()
+
+
+class RangeGeneratorCLI(pki.cli.CLI):
```

Comment:
The `pki-server <subsystem>-range-generator-*` name is probably not quite accurate since this command can be used to show RSNv3 config or migrate to RSNv3 which doesn't have ranges. How about something like `pki-server <subsystem>-id-generator-*`?